typescript/infra/scripts/funding/fund-relayers-from-deployer.ts

```diff
@@ -43,11 +47,15 @@ async function fundRelayer(
   const desiredBalanceEther = ethers.utils.parseUnits(desiredBalance, 'ether');
   const delta = desiredBalanceEther.sub(currentBalance);
 
-  if (delta.gt(MIN_DELTA)) {
+  const minDelta = desiredBalanceEther
+    .mul(MIN_DELTA_NUMERATOR)
+    .div(MIN_DELTA_DENOMINATOR);
+
+  if (delta.gt(minDelta)) {
     console.log(
-      `sending ${relayer.chainName} relayer ${ethers.utils.formatEther(
-        delta,
-      )}...`,
+      `sending ${relayer.chainName} relayer ${
```

Comment:
Optional: We should probably consider having structured logs her instead of just strings